client.go

```diff
@@ -184,8 +184,9 @@ type DNSProvider interface {
 
 // PipelinesProvider manages lifecyle of CI/CD pipelines used by a Function
 type PipelinesProvider interface {
-	Run(context.Context, Function) error
+	Run(context.Context, Function, bool) error
```

Comment:
I think @lkingland will not like this.
Maybe custom pipeline can exist rather that new implementer of the interface?